src/components/common/MediaCard.js

```diff
@@ -87,7 +87,7 @@ export const MediaCard = props => {
 
       {showActions && (
         <CardActions disableSpacing>
-          <ClearButton id={id} />
+          <ClearButton spaceId={id} />
```

Comment:
Why this change?

Reply:
ClearButton was defined to take spaceId as prop, so it was previously a bug. I'll also change the other buttons so that they all take `spaceId` and not `id`.